android/app/src/main/res/values-af/strings.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like that Weblate has synchronized the list of languages across components. That's fine.

Reply:
Actually, let me disable that plugin.